### ISSUE TITLE
Stata defaults

### ISSUE THREAD

**Issue by evanmorier** [atahk/bucky#1]
Alex, thanks for this package! I noticed in the documentation for vcovCR that "The default type of "CR" uses the same adjustment as 'Stata'." Based on [this Stata documentation](https://www.stata.com/manuals13/u20.pdf) (p. 54, "Technical Note"), though, it looks like that default is only for MLE models. For OLS models the default adjustment is CR1 (HC1). With a basic example I replicate Stata's errors using the CR1 option in robustify. Happy to give example data and R/Stata code if it's helpful!

**Comment by briatte** [comment]
Hello,

I have a related question -- would it be possible to document the package, in order to help users (like me) who are interested in using the package to replicate results obtained with Stata?

**Comment by atahk** [comment]
Yes, absolutely. There should be basic documentation for each function already, but I'll try to add more details and perhaps a tutorial. And I'll correct the OLS default shortly. Thanks for catching that, @evanmorier.

**Comment by atahk** [comment]
I have updated the `robustify` command to mirror the Stata defaults (although I believe vcovHC differs from Stata's calculation of HC0 slightly, this is the closest I think we can come using vcovHC; it should match exactly for HC1, as is now the default for `lm` models). I have updated the documentation to match.

I'm closing this issue, but feel free to open a new one if this isn't working properly or you'd like to suggest improvements to the documentation. Note that these changes are on GitHub now, but won't be in the version on CRAN until the next release.
